besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2825,6 +2825,14 @@ private TransactionPoolConfiguration buildTransactionPoolConfiguration() {
       txPoolConfBuilder.priceBump(Percentage.ZERO);
     }
 
+    if (getMiningParameters().getMinTransactionGasPrice().lessThan(txPoolConf.getMinGasPrice())) {
+      logger.warn(
+          "Forcing tx-pool-min-gas-price="
```

Comment:
is this preferable to flagging it as a misconfiguration?

Reply:
changed to adjust the value when only `min-gas-price` is set, and return and error when both are set, but `tx-pool-min-gas-price` > `min-gas-price`

Reply:
the updated code wants to be backward compatible, and to simplify the upgrade path, when users just want to set `min-gas-price=0`, without having to worry about the new option.